{"name": "test_add2[【类型】输入中文，给出提示信息2]", "status": "passed", "steps": [{"name": "步骤一：4和字开始计算，预期应等于参数大小超出范围，打印<ExceptionInfo for raises contextmanager>", "status": "broken", "statusDetails": {"message": "TypeError: '>' not supported between instances of 'str' and 'int'\n", "trace": "  File \"D:\\zj\\PycharmProjects\\study202208\\study0818\\test\\test_calculate.py\", line 52, in test_add2\n    result=self.cal.add(x,y)\n  File \"D:\\zj\\PycharmProjects\\study202208\\script\\calculator.py\", line 4, in add\n    if a > 99 or a < -99 or b > 99 or b < -99:\n"}, "start": 1660805207735, "stop": 1660805207735}], "attachments": [{"name": "log", "source": "0e678720-df59-4c16-9bb0-3103f00dd7b1-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "3d71f998-acea-454f-864d-57e91f35fc6f-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "bea02257-0845-4f51-9f11-f9c149c27735-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "x", "value": "4"}, {"name": "y", "value": "'字'"}, {"name": "expect", "value": "'参数大小超出范围'"}], "start": 1660805207734, "stop": 1660805207735, "uuid": "60ae092a-b11f-48e9-96c0-aed8da8f57f0", "historyId": "f18f3197389620d993b415e56fa662e5", "testCaseId": "c1fd4152c4894ab47d6f7448dc1c5bcf", "fullName": "study0818.test.test_calculate.TestAdd#test_add2", "labels": [{"name": "feature", "value": "加法计算模块"}, {"name": "story", "value": "加法报错用例"}, {"name": "tag", "value": "P2"}, {"name": "parentSuite", "value": "study0818.test"}, {"name": "suite", "value": "test_calculate"}, {"name": "subSuite", "value": "TestAdd"}, {"name": "host", "value": "LAPTOP-BEO6NH7O"}, {"name": "thread", "value": "7552-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "study0818.test.test_calculate"}]}